models/project/board.go

```diff
@@ -316,3 +329,12 @@ func UpdateBoardSorting(ctx context.Context, bs BoardList) error {
 		return nil
 	})
 }
+
+func GetColumnsByIDs(ctx context.Context, columnsIDs []int64) (BoardList, error) {
```

Comment:
I recommend passing a `projectID int64`, and then using `And(project_id = projectID)` to prevent any unintended access.

Reply:
f280c9b